src/index.js

```diff
@@ -18,15 +19,23 @@ export default dataLoading;
 const showTask = (i) => {
   const li = document.createElement('li');
   const inputCheckbox = document.createElement('input');
+  const paragraph = document.createElement('input');
   inputCheckbox.setAttribute('type', 'checkbox');
   if (tasks[i].completed === false) {
     inputCheckbox.removeAttribute('checked');
   } else {
     inputCheckbox.setAttribute('checked', 'checked');
   }
-  const paragraph = document.createElement('input');
+  inputCheckbox.addEventListener('change', () => {
+    if (inputCheckbox.checked) {
+      paragraph.classList.add('extra');
+    }
+    tasks[i].completed = inputCheckbox.checked;
+    localStorage.setItem('datas', JSON.stringify(tasks));
+    dataLoading();
+  });
   paragraph.setAttribute('type', 'text');
-  paragraph.setAttribute('id', 'taskField');
+  paragraph.setAttribute('id', `taskField${i}`);
```

Comment:
- [ ] Your app works as expected 😉 However, i noticed that there is a prefilled text `new task` in the input field and whenever you want to add a new task, you have to clear it befor you can type your task. This isn't ideal for user experience and also from the exanmple video you were given, that wasn't the recommended behaviour of the app so i will suggest you change it so that the input area will be empty and after new tasks are added, it goes back to being empty again 💯 
- [ ] I will suggest that you can make the text `new task` a placeholder if you still want to keep it there and not actual text in the input field as it currently is 👍. If not you can just remove it completely 👍 
- Video Reference

https://github.com/MenendezON/WP-To-Do-list/assets/102699958/b3be64da-9eb3-4814-be40-22c41d5d8fbf